### ISSUE TITLE
jQuery license is missing in metadata

### ISSUE THREAD

**Issue by mgorny** [sphinx-contrib/jquery#17]
(semi-related to #11)

Please consider listing the license for jQuery in `license.text`, since the file is included in the installed package.

**Comment by apteryks** [comment]
Yes, this package should mention both BSD-2 (and note just BSD, which is interpreted as BSD-0 by PyPI) and Expat (MIT) licenses.
